mlflow/tracking/_tracking_service/client.py

```diff
@@ -80,7 +81,38 @@ def get_metric_history(self, run_id, key):
 
         :return: A list of :py:class:`mlflow.entities.Metric` entities if logged, else empty list
         """
-        return self.store.get_metric_history(run_id=run_id, metric_key=key)
+
+        # NB: Paginated query support is currently only available for the RestStore backend.
+        # FileStore and SQLAlchemy store do not provide support for paginated queries and will
+        # raise an MlflowException if the `page_token` argument is not None when calling this
+        # API for a continuation query.
+
+        max_results = GET_METRIC_HISTORY_MAX_RESULTS
```

Comment:
Do we need this assignment? Can we pass `GET_METRIC_HISTORY_MAX_RESULTS` to `get_metric_history` directly?